server/http_handler.go

```diff
@@ -150,7 +145,7 @@ func (t *httpHandlerTool) getMvccByHandle(tableID, handle int64) (*kvrpcpb.MvccG
 
 func (t *httpHandlerTool) getMvccByStartTs(startTS uint64, startKey, endKey []byte) (*kvrpcpb.MvccGetByStartTsResponse, error) {
 	for {
-		curRegion, err := t.regionCache.LocateKey(t.bo, startKey)
+		curRegion, err := t.regionCache.LocateKey(tikv.NewBackoffer(context.Background(), 500), startKey)
```

Comment:
I think we could call `tikv.NewBackoffer(context.Background(), 500)` outside the for loop, and 500ms is too small, maybe just retry one time.

Reply:
Done.